roles/kubernetes-apps/helm/tasks/gen_helm_tiller_certs.yml

```diff
@@ -1,110 +0,0 @@
----
```

Comment:
fine to remove this file because of `name: Helm | Set up TLS` which is only for v2.